src/malloc.c

```diff
@@ -592,7 +592,7 @@ MAX_RELEASE_CHECK_RATE   default: 4095 unless not HAVE_MMAP
 #define FOOTERS 0
 #endif  /* FOOTERS */
 #ifndef ABORT
-#define ABORT  abort()
+#define ABORT  shmem_runtime_abort(1, "OpenSHMEM exited in error")
```

Comment:
Update string to "Sandia OpenSHMEM memory allocation error".  Use name from config.h.